pkg/network/ebpf/c/protocols/http2/decoding.h

```diff
@@ -504,4 +407,115 @@ int socket__http2_frames_parser(struct __sk_buff *skb) {
     return 0;
 }
 
+/* TLS */
+
+SEC("uprobe/http2_tls_entry")
+int uprobe__http2_tls_entry(struct pt_regs *ctx) {
+    const u32 zero = 0;
+    tls_dispatcher_arguments_t *info = bpf_map_lookup_elem(&tls_dispatcher_arguments, &zero);
+    if (info == NULL) {
+        log_debug("[http2_tls_entry] could not get tls info from map");
+        return 0;
+    }
+
+    http2_tls_state_key_t key = { 0 };
+    key.tup = info->tup;
+    key.length = info->len;
+    http2_tls_state_t *state = bpf_map_lookup_elem(&http2_tls_states, &key);
+
+    if (!state) {
+        struct http2_frame header = { 0 };
+        bool relevant = is_relevant_frame_tls(info, &header);
+
+        bool header_only = info->len == HTTP2_FRAME_HEADER_SIZE;
+        if (!header_only && !relevant) {
+            // The buffer seems to contain the whole frame, and the frame is not
+            // relevant: we skip it without storing some state.
+            goto exit;
+        }
```

Comment:
not sure if this condition is right
if we see get a full headers payload, then you'll drop it

while this logic fit into the scenario we've seen in go-tls grpc, does it true to grpc over python? grpc over java?